config/application.rb

```diff
@@ -120,5 +123,7 @@ class Application < Rails::Application
     Rails.application.routes.default_url_options[:locale] = I18n.locale
     config.i18n.default_locale = :en
     config.i18n.available_locales = %i[en cy] # overriden in test to allow en-GB for Faker
+
+    config.x.team_email = 'apply-for-legal-aid@digital.justice.gov.uk'
```

Comment:
Should this be in the open?  Does github get scraped for spam emails? Maybe it should be in the settings files, it will mean duplication but at least it would be encrypted

Reply:
Good point.